app/src/main/res/values-en/strings.xml

```diff
@@ -902,9 +902,9 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_fireHydrant_type_wall">Wall</string>
     <string name="quest_fireHydrant_type_pond">Pond</string>
 
-    <string name="quest_footwayPartSurface_title">What’s the surface of the footway here?</string>
+    <string name="quest_footwayPartSurface_title">What’s the surface of the footpath here?</string>
 
-    <string name="quest_fuelSelfService_title">Can you pump gas yourself here?</string>
+    <string name="quest_fuelSelfService_title">Does this gas station have self-service?</string>
```

Comment:
When I brought this up in Slack a while back, I think there was a concern about “self-service” being an unfamiliar term for translators. A comment here can clarify that we indeed mean a situation where patrons can pump gas for themselves.

Reply:
Added a comment above this element.